modules/json/src/main/java/com/devonfw/module/json/common/base/ObjectMapperFactory.java

```diff
@@ -136,6 +140,11 @@ public ObjectMapper createInstance() {
       mapper.setSubtypeResolver(subtypeResolver);
     }
 
+    mapper.setSerializationInclusion(Include.NON_NULL);
+    mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
+    mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
Can we add a comment above:
```
// ignore unkown properties in JSON to prevent errors
// e.g. when the service has been updated/extended but the calling REST client is not yet updated
// see https://github.com/devonfw-wiki/devon4j/wiki/guide-service-layer#versioning